code/gradle.properties

```diff
@@ -20,5 +20,5 @@ appName = no.bouvet.app.imagelightbox
 displayName = Image lightbox
 vendorName = Bouvet Norge AS
 vendorUrl = http://www.bouvet.no
-xpVersion = 7.9.0
+xpVersion = 7.14.0
```

Comment:
Regarding the note I made above, perhaps we shouldn't really update XP version so high?

Reply:
We don't have to, of course. Let's try 7.11 then, to have som progress....